pkg/metadata/inventories/inventories.go

```diff
@@ -88,6 +88,7 @@ const (
 	AgentLogsTransport                 AgentMetadataName = "logs_transport"
 	AgentCWSEnabled                    AgentMetadataName = "feature_cws_enabled"
 	AgentProcessEnabled                AgentMetadataName = "feature_process_enabled"
+	AgentContainersEnabled             AgentMetadataName = "feature_containers_enabled"
```

Comment:
Container is a whole product so what about:

```suggestion
	AgentContainerProcessesEnabled             AgentMetadataName = "feature_container_processes_enabled"
```

Reply:
That makes sense, I went ahead and fixed it